packages/components/src/sandbox/index.js

```diff
@@ -242,3 +243,5 @@ export default function Sandbox( {
 		/>
 	);
 }
+
+export default withFilters( 'editor.Sandbox' )( Sandbox );
```

Comment:
The `editor` part here refers to the package, so this is incorrect.

Reply:
Yes, you're right. I didn't sure about the name neither. We can change it in case we consider this approach valid to go. 👍